### ISSUE TITLE
feature request: bold font theme variant

### ISSUE THREAD

**Issue by niklasmtj** [catppuccin/zed#14]
First of all, thank you very much for maintaining this repository! I really enjoy the use of catppuccin (especially frappé) after using One Dark Pro for aaaages. 

What I would like to ask if it is possible to also add a bold font-weight variant. There is a variant of One Dark Pro in VSCode that supports e.g. bold function names. Since I have some kind of color impairment this helps me seeing those a lot faster. 
I forked the repository and put the font-weight 800 to the function part of the theme:
```json
"function": {
  "color": "#8caaee",
  "font_style": null,
  "font_weight": 800
},
```

Example in catppuccin frappé font-weight 800:
<img width="540" alt="catppuccin frappé with 800 bold function names" src="https://github.com/catppuccin/zed/assets/22395665/3c0b3054-1389-4d2d-8768-1532659790dc">

I also tried it with font-weight 700 to match it with the other attributes like `enum` or `title`:
<img width="521" alt="catppuccin frappé with 700 bold function names" src="https://github.com/catppuccin/zed/assets/22395665/991c7911-5d14-4db3-bd64-cc1d8dc437f3">

Update: One idea that I just got is using the `italic` version as a starting point and change `font-style: "italic"` to `font-style: null` and `font-weight: null` to `font-weight: 700` on those places. This leads to:
<img width="549" alt="all font-weight 700 everything" src="https://github.com/catppuccin/zed/assets/22395665/08058830-1c12-42ef-9de2-34f14a62c085">

What do you think about adding such kind of theme? If you're interested I can even open a PR for it, since I changed it locally already.

Thanks and I'm happy to hear your feedback.

**Comment by tecandrew** [comment]
hey @niklasmtj! [we discussed this a bit](https://github.com/catppuccin/zed/issues/6#issuecomment-2030256223) when I shipped the `non-italics` variant of the themes and decided that we're not going to do this since there aren't really these variants on other Catppuccin ports.

I'd like to suggest of doing the alternative of just _overriding_ the theme(s) using Zed's `settings.json` file. Add the following snippet to have all Functions be bold in the way you'd like 😄 

```json
  "experimental.theme_overrides": {
    "syntax": {
      "function": {
        "color": "#8caaee",
        "font_style": null,
        "font_weight": 800
      }
    }
  }
```
